docs/source/tutorials/GEDI/GEDI_L4A_Example.ipynb

```diff
@@ -0,0 +1,584 @@
+{
```

Comment:
<p>Is this GeoJSON file in a specific shared location? If so, replace <code>{username}</code> with a <em>specific</em> username, since other uses will <em>not</em> have this file in their <em>own</em> shared folder.    </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/133/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>This is shared under my bucket associated with my username. I'll update the aoi with my username and also leave a comment for users on how they can have their own aoi URL.</p>